react/features/base/i18n/functions.tsx

```diff
@@ -24,7 +24,7 @@ export async function changeLanguageBundle(language: string, url: string) {
  * @returns {Component} The React Component which wraps {@link component} and
  * enables translations in it.
  */
-export function translate(component: any) {
+export function translate<P extends WithTranslation>(component: React.ComponentType<P>) {
```

Comment:
This change doesn't belong in here, can you split it out please?

Reply:
I don't understand